apps/crn-frontend/src/network/working-groups/api.ts

```diff
@@ -42,3 +48,27 @@ export const getWorkingGroup = async (
   }
   return resp.json();
 };
+
+export const createWorkingGroupResearchOutput = async (
```

Comment:
Isn't this a duplicate of the existing API? We should probally just use that one